cvat-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvat-ui",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Wasn't the version in ``package-lock.json`` increased automatically with ``npm version minor``?

Reply:
@bsekachev don't track `package-lock.json` due to local packages mismatch. I will take this into account.